src/nni_manager/common/utils.ts

```diff
@@ -154,6 +154,10 @@ function parseArg(names: string[]): string {
 function getMsgDispatcherCommand(tuner: any, assessor: any): string {
     let command: string = `python3 -m nni --tuner_class_name ${tuner.className}`;
 
+    if (process.env.VIRTUAL_ENV) {
+        command = path.join(process.env.VIRTUAL_ENV + 'bin/') +command;
+    }
+
     if (tuner.classArgs !== undefined) {
```

Comment:
it is not good to modify code here

Reply:
I'm just still testing travis. Not ready